Sources/Suggestions/SuggestionResult.swift

```diff
@@ -24,9 +24,9 @@ public struct SuggestionResult: Equatable {
         SuggestionResult(topHits: [], duckduckgoSuggestions: [], localSuggestions: [])
     }
 
-    private(set) public var topHits: [Suggestion]
-    private(set) public var duckduckgoSuggestions: [Suggestion]
-    private(set) public var localSuggestions: [Suggestion]
+    public var topHits: [Suggestion]
+    public var duckduckgoSuggestions: [Suggestion]
+    public var localSuggestions: [Suggestion]
```

Comment:
Wouldn't it be better to make this `let`s instead rather than effectively making this mutable?  I should have done that when I made the last changes, sorry.

In the macOS code that is overwriting these it would be better to re-construct the result via the constructor instead.